cypress/e2e/immutableDatabaseTests/dropdown.ts

```diff
@@ -431,4 +430,10 @@ describe('Dropdown test', () => {
     //         .click()
     // });
   });
+  describe('Log out', () => {
+    it('Should be able to log out through the dropdown button', () => {
+      cy.get('[data-cy=dropdown-logout-button]').should('be.visible').click();
```

Comment:
This isn't going to work with my upcoming Cypress 12 PR :( . You had no way of knowing that, and we have this pattern all over -- I've been changing them.

I can fix it after you merge, or if you want to get it ready in advance...

In Cypress 12, different tests are isolated by default. This test currently assumes you are

1. On a page, carried over from the previous test
2. Logged in

In Cypress 12
1. You won't be on a page because you need to explicitly navigate to one
2. You won't be logged in (local storage, with the token, is wiped between tests).

There are two possible fixes:

1. Get rid of lines 433 and 434 (and their closing braces), so that the previous page and logged in token "carry over".
2. Keep lines 433 and 434, but do a `cy.visit` and the call to set the user token.

I will probably just do 1 because it's easier; if you feel strongly and want to keep the log out as a separately named test, then could you please do 2.

Reply:
I'll change it!